src/Common/tests/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netcoreapp-Unix;
       netcoreapp-Windows_NT;
+      netfx;
```

Comment:
This might be too large of a change, so we could remove the string.Contains asserts from the span tests. We already test the string.Contains overloads in the existing string tests.

Alternatively, we could try and split the netcoreapp specific tests just like in System.Runtime.Tests (StringTests.netcoreapp.cs is only included for netcoreapp/uapaot):
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/tests/System.Runtime.Tests.csproj#L209

Doing so, you won't need to add an ifdef in code.

I believe you also have to add property groups in Common.Tests.csproj:
```csproj
  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Debug|AnyCPU'" />
  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Release|AnyCPU'" />
```

cc @tarekgh, what are your thoughts on this?

Reply:
The other idea is to write the test marked with skip on platform attribute. so you'll have 2 tests, one skips NetFx platform and tests the net core behavior. the other test will skip the net core and test NetFx behavior. 

If you didn't like this idea, then I would go with splitting the test source file into NetFx one and netcore one and conditionally include the appropriate source files in the csproj (this is as @ahsonkhan suggested above)

Reply:
> Alternatively, we could try and split the netcoreapp specific tests

You mean a partial class with "only netcoreapp working tests"(new Contains tests for now)?

> I believe you also have to add property groups in Common.Tests.csproj: ...

I don't understand...if i try to compile with netfx switch Common tests fails "by design"(without adding new tests, try `msbuild /t:rebuildandtest /v:m /p:TargetGroup=netfx` on clean repo)...i mean seems that Common tests are not able to run on netfx(es: some issue with `PathInternal.cs`)

Reply:
> The other idea is to write the test marked with skip on platform attribute. so you'll have 2 tests, one skips NetFx platform and tests the net core behavior. the other test will skip the net core and test NetFx behavior.

I don't think that will work since building the test would fail (it's a compile time error since the Contains overload doesn't exist on netfx).

> You mean a partial class with "only netcoreapp working tests"(new Contains tests for now)?

Yes.

> i mean seems that Common tests are not able to run on netfx(es: some issue with

I am not familiar enough with all the common tests. **For the purposes of this PR, I think removing the string.Contains components of the test (and removing the netfx configs) is probably easiest.**

If you prefer to keep debugging, see if we can move the PathInternal related test files that won't compile for netfx under a !netfx condition as well.

@JeremyKuhne, do you know what we need to change to get the Common tests building for netfx?
```text
12:51:52 Tests\System\PasteArgumentsTests.cs(22,56): error CS0117: 'PasteArguments' does not contain a definition for 'Paste' [D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\tests\Common.Tests.csproj]
12:51:52 Tests\System\PasteArgumentsTests.cs(32,56): error CS0117: 'PasteArguments' does not contain a definition for 'Paste' [D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\tests\Common.Tests.csproj]
12:51:52 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\src\CoreLib\System\IO\PathInternal.cs(17,35): error CS0103: The name 'IsDirectorySeparator' does not exist in the current context [D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\tests\Common.Tests.csproj]
12:51:52 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\src\CoreLib\System\IO\PathInternal.cs(22,106): error CS0103: The name 'IsDirectorySeparator' does not exist in the current context [D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\tests\Common.Tests.csproj]
```